### ISSUE TITLE
Error when tapping play icon for any station

### ISSUE THREAD

**Issue by retro486** [jaywink/SomaFM#1]
The player interface starts to load then dies with a blank screen saying "Oops, something went wrong." and the option to reload but then the same error happens.

Nexus 4 OTA-14 Stable

**Comment by retro486** [comment]
I should add when I go to somafm in the browser it works. Doesn't play in the background of course but doesn't crash so it's not a media/codec issue it seems.

**Comment by jaywink** [comment]
Yeah it's probably an app issue - or maybe the app just needs to be rebuilt against newer shipped dependencies. Unfortunately I don't have a working Ubuntu phone at the moment, and I'm hesitant to rely on any emulator to ship an app.

I've unpublished the app from the store for now. Are you a dev, want to try fix it? Or someone else? :) I'd be happy to grant collaborator access to maintain the app in the store - or just keep it unpublished so someone can push a fixed version, Bumping a fix to the same app would allow updates to go through though.
